hypha/static_src/javascript/review-form-score.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+    "use strict";
+
+    // grab all the selectors
+    let filtered_selectors;
+    const selectors = Array.prototype.slice.call(
+        document.querySelectorAll("select")
+    );
+
+    if (selectors.length > 1) {
+        document.querySelector(".form--score-box").style.display = "block";
+
+        // remove recommendation select box from array
+        filtered_selectors = selectors.filter(
+            (selector) => selector[0].text !== "Need More Info"
```

Comment:
What happens if this string is translated. Can we remove the element in a less fragile way?

Reply:
I'm not even sure this code had any effect so I removed it.